docs/modules/ROOT/pages/explanations/decisions/api-design.adoc

```diff
@@ -377,6 +377,15 @@ Additional points we've agreed on:
 * Exposing only needed fields, add more as needed
 * Sensible defaults, minimize required parameters.
 
+To minimize confusion when querying the claims. The kind should also contain the provider as a pre-fix:
```

Comment:
IMHO that's much more explanation. When I read that in a few month I don't know anymore what's meant with "To minimize confusion".

Reply:
I've added more in-depth explanation in the `rationale` section of the decision.